src/client/extensionActivation.ts

```diff
@@ -137,5 +131,11 @@ async function activateLegacy(
     deprecationMgr.initialize();
     context.subscriptions.push(deprecationMgr);
 
+    // "activate" everything else
+
+    const manager = serviceContainer.get<IExtensionActivationManager>(IExtensionActivationManager);
+    context.subscriptions.push(manager);
+    const activationPromise = manager.activate();
```

Comment:
This needed to be moved till later so the NotebookExtensibility class would be created in time for the survey banner to initialize.

Reply:
No, this needs to be moved back. this needs to happen before `datascience.activate`
The assumption today is that all IExentionActiaftion happens before everything else
However now this happens after DS code, that could break other areas.

Reply:
> This needed to be moved till later so the NotebookExtensibility class would be created in time for the survey banner to initialize.

Sorry, but don't understand the issue here.

Reply:
Oh never mind, this wasn't necessary either. I was having some trouble resolving the INotebookExtensiblity from the service container, but I think doing addBinding fixed it. Reverting this change